app/views/events/index.html.erb

```diff
@@ -104,6 +109,20 @@
             </div>
           <% end %>
         </div>
+         <!-- Add to calendar -->
+          <% if event.event_registrations.where(email: current_user.email).exists? %>
+          <div class="mt-4 pt-3">
+            <div class="text-sm font-semibold text-gray-600 mb-1">Add to calendar</div>
+            <div class="text-xs">
+              <%= link_to "Apple", "https://calndr.link/e/Hjs8zdcx1s?s=apple", class: "text-blue-600 hover:text-blue-800" %> •
```

Comment:
these are hard-coded links for one event. we'll need to dynamically make links for each event. (this is my fault for the ticket not being clear enough!)